src/background/services/openPayments.ts

```diff
@@ -694,7 +695,7 @@ export class OpenPaymentsService {
             },
             {
               type: 'outgoing-payment',
-              actions: ['create'],
+              actions: ['create', 'read'],
```

Comment:
This is the breaking change, as already users who have already connected extension will not have this permission in their tokens.
Need to handle this in `outgoingPayment.get()` by checking for "Inactive token" error, to make this a non-breaking change.